README.md

```diff
@@ -7,66 +7,47 @@
 
 ## Contributing
 
-We welcome Semgrep rule contributions directly to this repository! Since this repo is maintained by r2c, there are some extra benefits-for example, if there are bug reports for your rule, we’ll take responsibility to help fix it. If you are submitting to the semgrep-rules repo (rather than your own, separate repository as mentioned above) we’ll ask you to make r2c a joint owner of your contributions. While you still own copyright rights to your rule, joint ownership allows r2c to license these contributions to other [Semgrep Registry](https://semgrep.dev/r) users pursuant to the LGPL 2.1 under the [Commons Clause](https://commonsclause.com/). Check out the [Contributing Guidelines](/CONTRIBUTING.md) to get started.
-
-You can also contact us at support@r2c.dev to make Semgrep rule contributions. We will import your rules for everyone to use!
-
-### Rulesets
-
-Rulesets -- combined sets of rules from the Semgrep registry -- are organized in a private repository. If you want to modify existing sets or create your own, please contact us at support@r2c.dev.
-
-If you have more questions, please see the [FAQ section in the Semgrep docs](https://semgrep.dev/docs/faq).
-
-## Help
+We welcome Semgrep rule contributions directly to this repository! If you are submitting to the `semgrep-rules` repository we’ll ask you to make r2c a joint owner of your contributions. While you still own copyright rights to your rule, joint ownership allows r2c to license these contributions to other [Semgrep Registry](https://semgrep.dev/r) users pursuant to the LGPL 2.1 under the [Commons Clause](https://commonsclause.com/). Full [license details](https://github.com/returntocorp/semgrep-rules/blob/develop/LICENSE) here.
 
-Join [Slack](https://r2c.dev/slack) for the fastest answers to your questions! Or contact the team at [support@r2c.dev](mailto:support@r2c.dev).
+:::info
+To contribute, find details about contributing in the **[Contributing to Semgrep rules](https://semgrep.dev/docs/contributing/contributing-to-semgrep-rules-repository/)** documentation.
+:::
 
-### Rule Namespacing
-
-The namespacing format for contributing rules is `<language>.<framework>.<category>.$MORE`. If a `framework` isn't applicable, use `lang` instead.
-
-`category` is one of:
-- security
-- correctness
-- best-practice
-- maintainability
-- performance
+You can also contact us at support@r2c.dev to make Semgrep rule contributions. We will import your rules for everyone to use!
 
-If a `security` rule is discouraging the use of a bad pattern (such as formatted SQL strings), we recommend appending `audit` to your namespace. This distinguishes it from a `security` rule that is specifically aiming to detect a vulnerability.
+## Additional information
 
-<p align="center">
-    <img src="https://web-assets.r2c.dev/semgrep-live-namespacing.png" alt="semgrep.live rule namespace" width="500" />
-</p>
```

Comment:
Maybe I should move this screenshot to our [Contributing docs](https://semgrep.dev/docs/contributing/contributing-to-semgrep-rules-repository/)? @milanwilliams

Reply:
This screenshot is definitely outdated, I think it's okay to remove it completely! 😄